README.md

```diff
@@ -105,33 +127,10 @@ template:
                 {%- set ns.days = daysLeft -%}
               {%- endif -%}
             {%- endfor -%}
-            {%- for birthday in states.birthdays -%}
-              {%- set daysLeft = birthday.state | int -%}
-              {%- if daysLeft == ns.days -%}
-                {%- set ns.ages = ns.ages + [birthday.attributes.age_at_next_birthday] -%}
-              {%- endif -%}
-            {%- endfor -%}
-
-            {{ns.ages | join(', ')}}
-          birthday_message: >
-            {%- set ns = namespace(days=365, messages=[]) -%}
-            {%- for birthday in states.birthdays -%}
-              {%- set daysLeft = birthday.state | int -%}
-              {%- if daysLeft < ns.days -%}
-                {%- set ns.days = daysLeft -%}
-              {%- endif -%}
-            {%- endfor -%}
-            {%- for birthday in states.birthdays -%}
-              {%- set daysLeft = birthday.state | int -%}
-              {%- if daysLeft == ns.days -%}
-                {%- set ns.messages = ns.messages + [birthday.attributes.friendly_name + ' turns ' + (birthday.attributes.age_at_next_birthday | string) + ' today!'] -%}
-              {%- endif -%}
-            {%- endfor -%}
```

Comment:
This should be left as is in master-branch (so that it supports new template style in HA)